frontend/packages/dev-console/integration-tests/features/common/project-creation.feature

```diff
@@ -0,0 +1,10 @@
+Feature: OpenShift Namespaces
+              As a user, I want to create the namespace to group and isolate related objects
+
+        @smoke
+        Scenario: Create the namespace
+            Given user is at developer perspective
+             When user selects the Create Project option from Projects dropdown on top navigation bar
+              And user enters project name as "aut-project" in Create Project modal
+              And user clicks Create button present in Create Project modal
+             Then modal will get closed
```

Comment:
This an indentation problem?

Reply:
nope, this is expected. Based on vs code plugin "Siarhei Kuchuk". This indentation appears

Reply:
Most of your other `.feature` files don't have this indentation, seems odd... but hey, right now QE is the only one maintaining these... if you're good with the indents, who am I to say it's odd 🙂